build/cli/src/arg/gui.rs

```diff
@@ -2,43 +2,31 @@ use enso_build::prelude::*;
 
 use crate::arg::BuildJob;
 use crate::arg::Source;
-use crate::arg::WatchJob;
 use crate::source_args_hlp;
-use crate::IsWatchableSource;
 
 use clap::Args;
 use clap::Subcommand;
 use enso_build::project::gui::Gui;
-use enso_build::project::wasm::Wasm;
 
 
 
 source_args_hlp!(Gui, "gui", BuildInput);
 
-impl IsWatchableSource for Gui {
-    type WatchInput = WatchInput;
-}
-
-#[derive(Args, Clone, Debug, PartialEq)]
-pub struct BuildInput {
-    #[clap(flatten)]
-    pub wasm: Source<Wasm>,
-}
-
-#[derive(Args, Clone, Debug, PartialEq)]
-pub struct WatchInput {
-    #[clap(flatten)]
-    pub wasm: <Wasm as IsWatchableSource>::WatchInput,
-}
+#[derive(Args, Clone, Copy, Debug, PartialEq)]
+pub struct BuildInput {}
 
 #[derive(Subcommand, Clone, Debug, PartialEq)]
 pub enum Command {
     /// Builds the GUI from the local sources.
     Build(BuildJob<Gui>),
     /// Gets the GUI, either by compiling it from scratch or downloading from an external source.
     Get(Source<Gui>),
+    /// Runs the GUI's unit tests.
+    Test,
+    /// Run linter on the GUI's sources.
+    Lint,
     /// Continuously rebuilds GUI when its sources are changed and serves it using dev-server.
-    Watch(WatchJob<Gui>),
+    Watch,
```

Comment:
I think `rebuilds` is not quite the right word, as our watch (which internally runs `npm run dev`) serves local files without building (only dependencies may be optimized).

Reply:
> I think rebuilds is not quite the right word, as our watch (which internally runs npm run dev) serves local files without building (only dependencies may be optimized).

If I understand correctly, it "rebuilds" the GUI in terms of transpiling and bundling and catching any TS build errors? I don't think that browser can just run local files? Or am I wrong?

Reply:
You're right, I confused building with bundling.

Reply:
i don't think it catches TS build errors, nor bundles (but it does transpile of course.)

bundling would break HMR; typechecking would affect latency between editing, and the code getting updated in the browser